e2e/main-nav.spec.ts

```diff
@@ -7,7 +7,7 @@ test('test main navigation', async ({ page }) => {
   const navigationBar = page.getByTestId('main-navigation');
   for (const link of navLinks) {
     await navigationBar.getByRole('link', { name: link.name }).click();
-    await page.waitForTimeout(2 * 1000);
+    await page.waitForTimeout(2 * 4000);
```

Comment:
```suggestion
    await page.waitForTimeout(8000);
```
why, `2* 4000`, normally we use the milliseconds for easy format, in that case should be `8 * 1000`